testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/HostExcludesTestCase.java

```diff
@@ -208,24 +210,29 @@ private enum ExtensionConf {
         // to the internal mpExtensions Set defined on this class.
         // Don't add here extensions supplied only by the wildfly-preview-feature-pack because we are not tracking different releases
         // of wildfly preview. In such a case, add them to previewExtensions set defined below.
-        CURRENT(MAJOR, WILDFLY_27_0, null, getCurrentRemovedExtensions());
+        CURRENT(MAJOR, WILDFLY_27_0, Arrays.asList(
+                "org.wildfly.extension.micrometer"
+        ),
+                getCurrentRemovedExtensions()
+        );
 
         private static List<String> getCurrentRemovedExtensions() {
-            // TODO If we decide to remove these modules from WFP, uncomment this.
-            // See https://issues.redhat.com/browse/WFLY-16686
-            /*
             if (AssumeTestGroupUtil.isWildFlyPreview()) {
                 return Arrays.asList(
-                        "org.jboss.as.messaging",
-                        "org.jboss.as.jacorb",
-                        "org.jboss.as.jsr77",
-                        "org.jboss.as.web",
-                        "org.wildfly.extension.picketlink",
-                        "org.jboss.as.security"
+                        // This is moving from Preview to Standard in WF 28, so it's both being removed and added.
+                    "org.wildfly.extension.micrometer"
```

Comment:
@yersan @bstansberry, this feels hinky, but it works. If there's a more correct way, please chime in.